### ISSUE TITLE
multi folder project opens wrong (new) file

### ISSUE THREAD

**Issue by davy79** [amitab/atom-cscope#13]
Hi,

I was facing an issue when using a project with several folders in a specific setup.
Maybe related to #12 ?

My file structure:

```
root/
   project1/
   project2/
   common_components/
      comp1/
      comp2/
```

I'm using the "project manager" package to quickly jump between projects. and it allows to specify separate folders as 1 project. So I can select only `project1/` and the `common_components/`
So Atom is now listing 2 folders at the top level in the tree view. (which are also at the same level in the file system)

```
project1/
common_components/
   comp1/
   comp2/
```

generating a cscope DB puts cscope files in both `project1/` and `common_components/`

searching for a symbol yields the expected results and works fine.
however when navigating to a result, atom sometimes opens a file with the wrong path.

example:

currenty in a file in `project1/file1` searching for a symbol and navigating from the results to `common_components/comp1/file2` atom actually opens a file in:
`project1/comp1/file2`
Which does not exist...

The reverse however does work...
searching from `common_components/comp1/file2` and navigating to a result in `project1/file1` does open the correct file in `folder1/file1`.

Seems like an issue appending subfolders to root folders?

**Comment by amitab** [comment]
Yes it's relaed to #12 
I did not consider multiple projects while creating this package. I'll be fixing this soon.

**Comment by amitab** [comment]
Fixed in master c1b15b9bfd50c0e5063934f54ba3ddaad6ba7018
